tests/cross_cluster_live_migration/conftest.py

```diff
@@ -32,8 +64,478 @@ def remote_kubeconfig_export_path(request):
 
 
 @pytest.fixture(scope="session")
-def remote_admin_client(remote_kubeconfig_export_path):  # skip-unused-code
+def remote_admin_client(remote_kubeconfig_export_path):
     """
     Get DynamicClient for a remote cluster
     """
     return get_client(config_file=remote_kubeconfig_export_path)
+
+
+@pytest.fixture(scope="session")
+def remote_cluster_api_url(remote_admin_client):
+    """
+    Get the API URL of the remote cluster.
+    Returns the cluster API endpoint URL (e.g., https://api.cluster-name.example.com:6443)
+    """
+    api_url = remote_admin_client.configuration.host
+    LOGGER.info(f"Remote cluster API URL: {api_url}")
+    return api_url
+
+
+@pytest.fixture(scope="session")
+def remote_cluster_service_account(remote_admin_client):
+    with ServiceAccount(
+        client=remote_admin_client,
+        name="remote-cluster-service-account",
+        namespace="default",
+    ) as sa:
+        yield sa
+
+
+@pytest.fixture(scope="session")
+def remote_cluster_cluster_role_binding(remote_admin_client, remote_cluster_service_account):
+    with ClusterRoleBinding(
+        name="remote-cluster-cluster-role-binding",
+        cluster_role="cluster-admin",
+        client=remote_admin_client,
+        subjects=[
+            {
+                "kind": ServiceAccount.kind,
+                "name": remote_cluster_service_account.name,
+                "namespace": remote_cluster_service_account.namespace,
+            }
+        ],
+    ) as crb:
+        yield crb
+
+
+@pytest.fixture(scope="session")
+def remote_cluster_service_account_token_secret(
+    remote_admin_client, remote_cluster_service_account, remote_cluster_cluster_role_binding
+):
+    """
+    Create a secret to hold the service account token.
+    The secret depends on the cluster role binding to ensure proper permissions.
+    """
+    with Secret(
+        client=remote_admin_client,
+        name="remote-cluster-sa-token-secret",
+        namespace=remote_cluster_service_account.namespace,
+        annotations={
+            "kubernetes.io/service-account.name": remote_cluster_service_account.name,
+        },
+        type="kubernetes.io/service-account-token",
+    ) as secret:
+        yield secret
+
+
+@pytest.fixture(scope="session")
+def remote_cluster_auth_token(remote_admin_client, remote_cluster_service_account_token_secret):
+    """
+    Get the authentication token for the remote cluster.
+
+    Returns:
+        str: The authentication token for the current user session
+    """
+    return wait_for_service_account_token(secret=remote_cluster_service_account_token_secret)
+
+
+@pytest.fixture(scope="session")
+def remote_hco_namespace(remote_admin_client):
+    return Namespace(client=remote_admin_client, name=py_config["hco_namespace"], ensure_exists=True)
+
+
+@pytest.fixture(scope="package")
+def hyperconverged_resource_scope_package_remote_cluster(remote_admin_client, remote_hco_namespace):
```

Comment:
please be consistent with the names - either use `remote_cluster` as prefix or as suffix in all places